contrib/resources/translations/README.md

```diff
@@ -0,0 +1,79 @@
+
+# General translation workflow
+
+## Preparing file with English strings
+
+First, you need up to date the file with English strings. The one present in
+this directory is most likely outdated; to produce the current one do the following:
+
+- In the configuration file, go to section `[dosbox]` and set`language = en`.
+- Run the most current version of DOSBox Staging.
+- Inside DOSBox execute command `config -wl en.txt` (you can supply host OS path
+  instead of bare `en.txt`).
+
+This will produce file with current strings to translate, use it as a reference.
+
+## Updating the translation
+
+To update translation just edit the relevant `*.lng` file with text editor,
+save the file as UTF-8 encoded, with Unix line endings.
+
+Depending on your host OS, use the correct script to normalize the output. Although
+DOSBox Staging should be able to handle non-normalized translation files, and
+normally text editors save normalized files already, it is still highly recommended
+to use the script to make sure no problem occurs.
+
+## String-specific tips
+
+### Format strings
+
+Many strings contain strange sequences starting from percent sign, like `%s` or `%02d`.
```

Comment:
I think this would be the best:

_Many strings contain strangely looking sequences starting from the percentage sign,
like %s or %02d_

The prinf format string special sequences always start with percentage sign.